tests/tick.test.mjs

```diff
@@ -455,8 +477,29 @@ describe('Tick', async () => {
       timestamp: distributionTimestamp,
     });
 
-    // assert no error tag
-    assertNoResultError(distributionTick);
+    // assert multiple messages are sent given the tick notice, epoch created notice and epoch distribution notice
+    assert.equal(distributionTick.result.Messages.length, 4); // cannot explain why this is 4, i'd expect it to be 3 messages (1 tick notice, 1 epoch created notice, 1 epoch distribution notice)
```

Comment:
added a comment here - we are getting one extra `Tick-Notice` at the end of the message array, unclear why. will spend some more time on it

Reply:
Whats the 4th message?

Reply:
it was mainly due to test setup, and some minor edge cases in how we create the first epoch. The latest commits have a few changes that address the issue.